nexus/db-model/src/zpool.rs

```diff
@@ -29,22 +30,36 @@ pub struct Zpool {
 
     // The physical disk to which this Zpool is attached.
     pub physical_disk_id: DbTypedUuid<PhysicalDiskKind>,
+
+    /// How much storage to prevent Crucible from allocating for regions on a
+    /// given pool.
+    control_plane_storage_buffer: i64,
```

Comment:
Should this be a `ByteCount` type? from `nexus/db-model/src/bytecount.rs`?

Reply:
yes! cc38e26d7